lib/ome/common/module.cpp

```diff
@@ -76,9 +76,9 @@ namespace
     // Global paths (not specific to any component)
     static RegisterModule bin
       ("bin",
-       "BIOFORMATS_BINDIR",
+       "OME_FILES_BINDIR",
```

Comment:
Despite your various comments, I'm still not understanding why, e.g., ome-common doesn't use `OME_COMMON`. I'm on the road for 3 days. I wouldn't hold up the work as it stands, but I would appreciate a chance to discuss when I get back.